fnConsensus/reactor.go

```diff
@@ -452,9 +455,12 @@ func (f *FnConsensusReactor) vote(fnID string, fn Fn, currentValidators *types.V
 	// Have we achieved Maj23 already?
 	aggregateExecutionResponse := voteSet.MajResponse(f.cfg.FnVoteSigningThreshold, currentValidators)
 	if aggregateExecutionResponse != nil {
+		if !f.isMessageHashEqual(fnID, voteMethodID) {
```

Comment:
Don't need to re-hash the message, just check that `bytes.Equal(f.state.Messages[fnID].Hash, response.Hash)` (need to make the change below first)